backtester/backtester.cpp

```diff
@@ -226,15 +251,55 @@ void runParamCombos(int i, vector<string>& rangeNames, vector<vector<double>>& r
 	}
 }
 
+void simulateDay(double& currentCash, unordered_map<string, double>& params, size_t& currentTime, double& currentQuote, Action& action,
+	unique_ptr<Algorithm>& algo, Day& day, bool& activePositions, Result& result, Trade& trade, size_t& stockUnits) {
+	for (size_t i = 0; i < day.quotes.size(); ++i) {
+		if (currentCash < params["minimumCash"]) {
```

Comment:
So if currentCash drops below minimumCash, we just stop processing anything for the whole day? I guess it's true that we don't want to buy more stocks, but we still might want to sell activePositions at a specific moment in the day provided by the algorithm right?